lib/pltr/v2/actions/books.js

```diff
@@ -5,19 +5,26 @@ import {
   REORDER_BOOKS,
   LOAD_BOOKS,
   ADD_BOOK_FROM_TEMPLATE,
+  EDIT_BOOK_IMAGE,
 } from '../constants/ActionTypes'
 import { book } from '../store/initialState'
 
-export function addBook() {
-  return { type: ADD_BOOK, book }
+export function addBook(title, premise, genre, theme) {
+  return { type: ADD_BOOK, book, title, premise, genre, theme }
 }
 
 export function addBookFromTemplate(templateData) {
   return { type: ADD_BOOK_FROM_TEMPLATE, book, templateData }
 }
 
-export function editBook(id, attributes) {
-  return { type: EDIT_BOOK, id, attributes }
+//  Breaking Changes
```

Comment:
Please document this change in the README.md file in `pltr`.
So at the end of the file at `lib/pltr/README.md`, add a new section that looks like this:
```
# Breaking Changes
## 2022-23-11
Old: `function editBook(id, attributes)`
New: `function editBook(id, title, premise, genre, theme)`